spec/features/cookie_preferences_spec.rb

```diff
@@ -10,7 +10,7 @@
     expect(page.current_path).to eq(root_path)
   end
 
-  scenario "a user accepts the cookies from valid path" do
+  xscenario "a user accepts the cookies from valid path" do
```

Comment:
is the 'x' here intentional or a stray typo @ekumachidi?

Reply:
It is intentional, I added it as I was validating the specs locally. Still working my way through the tests